.github/workflows/codeql.yml

```diff
@@ -74,8 +74,8 @@ jobs:
     #   If the Autobuild fails above, remove it and uncomment the following three lines.
     #   modify them (or add more) to build your code if your project, please refer to the EXAMPLE below for guidance.
 
-    - run: |
-        ./.github/workflows/codeql-buildscript.sh
+    - name: Build firmware
+      run: make list-boards | xargs -I'{}' make BOARD='{}' -j$(nproc)
```

Comment:
This won't work for building. And probably doesn't work for analysis either.

Reply:
CodeQL monitors the build process and creates a database to represent the hierarchical structure of code in the repository.